src/components/src/main/java/org/apache/jmeter/visualizers/backend/influxdb/HttpMetricsSender.java

```diff
@@ -48,11 +48,22 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Influxdb sender base on The Line Protocol. The Line Protocol is a text based
- * format for writing points to InfluxDB. Syntax : <measurement>[,<tag_key>=
- * <tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=
- * <field_value>] [<timestamp>] Each line, separated by the newline character,
- * represents a single point in InfluxDB. Line Protocol is whitespace sensitive.
+ * InfluxDB sender base on The Line Protocol.
+ * <p>
+ * The Line Protocol is a text based format for writing points to InfluxDB.
+ * Syntax:
+ * <pre>
+ * weather,location=us-midwest temperature=82 1465839830100400200
```

Comment:
@ham1 , could you please check https://github.com/apache/jmeter/blob/35247ae6fbbc699964ce2ead8a3d376ca7084a1d/src/components/src/test/java/org/apache/jmeter/visualizers/backend/influxdb/HttpMetricsSenderTest.java#L92 ?

It looks like the current implementation fails to put a comma, and it sends `measurementlocation=west size=10 ...`.
Is it a bug?

Reply:
This seems to be ok because the `String tags` inside `MetricTuple` always starts with a `,` but it does invite some refactoring to perhaps better represent the data? e.g. tags should be a `List` or `Map` rather than `String`?

See:
```
        tag.append(TAG_APPLICATION).append(application);
        tag.append(TAG_TRANSACTION).append("internal");
        tag.append(userTag);
```
Where `TAG_APPLICATION = ",application="` and `TAG_TRANSACTION = ",transaction="`

Reply:
>e.g. tags should be a List or Map rather than String

Can you please clarify which `tags` do you mean?

Reply:
I see now.
https://bz.apache.org/bugzilla/show_bug.cgi?id=60590 introduced `org.apache.jmeter.visualizers.backend.influxdb.InfluxdbMetricsSender#addMetric` in JMeter 3.1, so altering the API  would be a backward-incompatible change.

Reply:
I mean tag inside `InfluxdbMetricsSender.MetricTuple` could be a `Map` and then it gets converted to String inside `InfluxdbBackendListenerClient` instead of it assuming it's a String in the right format.

Reply:
That is still a part of the public API, so we need to figure out if anyone is using it before we can remove it.

Then we don't want to convert the same List/Map to String again and again (for every send), so String is probably required there.

We might want to add an extra constructor, however, I would refrain from that for now.

Reply:
I wasn't about to do it, as indeed performance might be an issue and backwards compatibility. But maybe some more docs would make it clearer what format the params should be in.

Reply:
I've updated the test to better reflect how it's used, i.e. with a `,`